src/main/java/org/symphonyoss/symphony/messageml/elements/TextArea.java

```diff
@@ -0,0 +1,98 @@
+package org.symphonyoss.symphony.messageml.elements;
+
+import org.commonmark.node.Node;
+import org.symphonyoss.symphony.messageml.exceptions.InvalidInputException;
+import org.symphonyoss.symphony.messageml.markdown.nodes.form.FormElementNode;
+import org.symphonyoss.symphony.messageml.util.XmlPrintStream;
+
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+public class TextArea extends FormElement {
+
+  public static final String MESSAGEML_TAG = "textarea";
+
+  private static final String PLACEHOLDER_ATTR = "placeholder";
+  private static final String REQUIRED_ATTR = "required";
+  private static final List<String>
+      VALID_VALUES_FOR_REQUIRED_ATTR = Arrays.asList("true", "false");
```

Comment:
We don't break lines if it's less than 80 or 100, we use wrap on 140 or so instead. 80 is too little.

Reply:
@cristiadu I'm using Symphony's code style configuration in IntelliJ IDEA, that's why it's formatted this way. Is my configuration out of date? Should I use another configuration instead?
Anyway, I'm going to double check the code and update it accordingly.

Reply:
Changed configuration for hard warp to 140.